index.js

```diff
@@ -237,15 +239,19 @@ const getRoundDetails = async (req, res, client, getCurrentRound, roundParam) =>
     res.setHeader('cache-control', 'no-store')
   }
 
+  await replyWithDetailsForRoundNumber(res, client, roundNumber)
+}
+
+const replyWithDetailsForRoundNumber = async (res, client, roundNumber) => {
   const { rows: [round] } = await client.query('SELECT * FROM spark_rounds WHERE id = $1', [roundNumber])
   if (!round) {
     return notFound(res)
   }
 
-  const { rows: tasks } = await client.query('SELECT * FROM retrieval_tasks WHERE round_id = $1', [roundNumber])
+  const { rows: tasks } = await client.query('SELECT * FROM retrieval_tasks WHERE round_id = $1', [round.id])
 
   json(res, {
-    roundId: roundNumber.toString(),
+    roundId: round.id.toString(),
```

Comment:
why did this change from `roundNumber` to `round.id`?

Reply:
I was re-using the code from https://github.com/filecoin-station/spark-api/pull/101 where this change was necessary.

It's not required for my PR, but I find it nicer that we are using the round number from the DB round row we fetched. If we ever need to make changes similar to #111 here again, then the code will be ready.